src/TicTacToe.elm

```diff
@@ -24,18 +25,25 @@ update : Msg -> Model -> Model
 update msg model =
   case msg of
     TakeTurn position ->
-      takeTurn position model
+      case model.playState of
+        InProgress model -> {playState = InProgress (takeTurn position model)}
```

Comment:
This variable name `model` is confusing:
The type `Model` is passed as an argument of `update` which is bound to the variable `model` on line 25.
Inside the model, there is a `playState` that has the `model` again when is `InProgress`? Is this an infinitely recursive data structure?
The definition of the type `PlayState` says that `InProgress` contains `Game`, not an instance of the type `Model`.